upgrade_test.py

```diff
@@ -356,6 +357,8 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
             self.upgradesstables_if_command_available(node)
             InfoEvent(message='upgrade_node - ended to "upgradesstables_if_command_available"').publish()
 
+        self.db_cluster.wait_all_nodes_un()
```

Comment:
can you elaborate on this change ? 

before we didn't wait for the node to be up normal, before moving to the next one ?

why do we need to check them all ? is the docs for upgrades were update with this addition ?

Reply:
I think we need better references for changing the procedure like that

Reply:
It's safer, we added the same thing in rolling restart.
Before raft, it was a bit faster so we harfly noticed it, but in general of course all should wait for UN.